src/extension/devtools/panel.html

```diff
@@ -8,6 +8,9 @@
       body {
         overflow: hidden;
       }
+      ::-webkit-scrollbar {
```

Comment:
In dark mode, `overflow: scroll` means white bars in the reserved space for the scroll bars (in FireFox, it respects the mode the browser is in which is visually much nicer as you can see in the screenshot).

It's possible to style the webkit scroll bar, but the APIs between webkit/non-webkit browsers are different. Hiding the scrollbar in webkit browsers is the past of least CSS for now.